### PR TITLE
Re-vamped event page

### PR DESCRIPTION
**Pull request by kmcnellis**
Added event banner, renamed course-banner to picture-banner, and added a comma list directive

Added event info page, reorganized edit links, linked event cards, clarified if user is in section event is assigned to.

### CODE REVIEW

File: client/app/instructor/studentsubmissions/studentsubmissions.html [hunk 1]
```diff
@@ -8,7 +8,9 @@
       <h1 class="page-header">Student Submissions</h1>
       <h3 ng-hide="submissions.length">No student submissions yet</h3>
       <h4 ng-hide="submissions.length">Create an <a href="/instructor/newevent">event</a> to get started</h4>
-      <submission-card class="col-md-6" ng-repeat='submission in submissions' data="submission" click="goToEvent(submission.sectionEvent)" short="false" />
+      <!-- <submission-card class="col-md-6" ng-repeat='submission in submissions' data="submission" click="goToEvent(submission.sectionEvent)" short="false" /> -->
```
Comment (seveibar) [inline client/app/instructor/studentsubmissions/studentsubmissions.html]: why is this removed?

**Comment by seveibar** [comment]
Awesome we needed this page (is this listed as an issue?)

I just had one comment on a changed line, functionality was when I pulled.

**Comment by seveibar** [comment]
(also the refactoring/restyling was nice :)

**Comment by kmcnellis** [comment]
No, I didn't see an issue for it

File: client/app/instructor/studentsubmissions/studentsubmissions.html (hunk 1 above)
Comment (kmcnellis) [inline client/app/instructor/studentsubmissions/studentsubmissions.html]: Mistake, was working on something else

**Comment by seveibar** [comment]
looks good to me
